src/Microsoft.SqlTools.ServiceLayer/QueryExecution/DataStorage/SaveAsCsvFileStreamWriter.cs

```diff
@@ -48,23 +48,57 @@ public SaveAsCsvFileStreamWriter(Stream stream, SaveResultsAsCsvRequestParams re
         /// </param>
         public override void WriteRow(IList<DbCellValue> row, IList<DbColumnWrapper> columns)
         {
-            string delimiter = ",";
+            char delimiter = ',';
```

Comment:
would we ever want to support multi-character delimiters?  I don't think we need to, but just asking.